F all tournaments page

heindrik opened the pull request:
added: tournaments page
removed: tournament listing in homepage
added: datatables on tournaments page. 


turbolinks gem needed to be downgraded.

Commit:
added tournament listings button to both navbar and home page, removed tournament listings from home page

Commit:
added datatables to tournament listings page

Commit:
added links to the tournaments in tournament listing page

Commit:
fixed reload bug for datatables

turbolinks gem needed to be downgraded.

heindrik commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Make "All Tournaments Page".](https://trello.com/c/akrpHIkW/47-make-all-tournaments-page)

Commit:
Removed conflicting bootstrap versions; fixed dropdown in menu

Commit:
fixed responsiveness of tournaments table

Commit:
added back all the javascript and removed google ajax from application view file

aakanshapangler commented:
I changed a few things around. I figured out that it was only the ajax breaking the table. Made the table responsive again. And also fixed the dropdown in the menu bar work again. :)

I think this is good to merge.

heindrik commented:
Hey yeah no problem. Yeah it was google ajax. The only reason I commented out the rest because it seems like those other files don't load. If you check the web console it will say that it cannot find those files. 

But thanks for fixing the rest. When your done testing let me know so I'll merge :)

Sent from my iPhone

> On Nov 26, 2016, at 11:59 AM, aakanshapangler <notifications@github.com> wrote:
> 
> I changed a few things around. I figured out that it was only the ajax breaking the table. Made the table responsive again. And also fixed the dropdown in the menu bar work again. :)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
